src/core/CorePool.ts

```diff
@@ -195,12 +200,14 @@ export class CorePool {
       // because each iteration of the while loop rounds, we can't optimize this code (relative to the smart contract)
       // by simply traversing to the next available tick, we instead need to exactly replicate
       // tickBitmap.nextInitializedTickWithinOneWord
-      ({ nextTick: step.tickNext, initialized: step.initialized } =
```

Comment:
Only format change here, please ignore.

Reply:
Sure but why is there such difference? Do I need some parameter setting with Prettier?